src/app/helptext/system/failover.ts

```diff
@@ -3,15 +3,16 @@ import globalHelptext from '../../helptext/global-helptext';
 
 export const helptext_system_failover = {
   dialog_initiate_failover_title: T("Initiate Failover"),
-  dialog_initiate_failover_message: T("A failover might cause temporary service interruption."),
+  dialog_initiate_failover_message: T("WARNING: A failover can \
```

Comment:
As Kris says, should not give false hope.
```suggestion
  dialog_initiate_failover_message: T("WARNING: A failover will \
```